packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -54,6 +54,22 @@ export interface ClusterProps {
    */
   readonly capacityProviders?: string[];
 
+  /**
+   * The cluster default capacity provider strategy. This takes the form of a list of CapacityProviderStrategy objects.
+   *
+   * For example
+   * [
+   *   {
+   *     capacityProvider: 'FARGATE',
+   *     base: 10,
+   *     weight: 50
+   *   }
+   * ]
+   *
+   * @default null
```

Comment:
This should describe the behavior when this value is not provided, not just list `null`.